apps/server/src/modules/video-conference/uc/video-conference-join.uc.ts

```diff
@@ -28,17 +28,17 @@ export class VideoConferenceJoinUc {
 
 		const user = await this.userService.findById(currentUserId);
 
-		const joinBuilder = new BBBJoinConfigBuilder({
-			fullName: this.videoConferenceService.sanitizeString(`${user.firstName} ${user.lastName}`),
-			meetingID: scope.id,
-			role,
-		}).withUserId(currentUserId);
-
 		const videoConference = await this.videoConferenceService.findVideoConferenceByScopeIdAndScope(
 			scope.id,
 			scope.scope
 		);
 
+		const joinBuilder = new BBBJoinConfigBuilder({
+			fullName: this.videoConferenceService.sanitizeString(`${user.firstName} ${user.lastName}`),
+			meetingID: scope.id + videoConference.salt,
```

Comment:
Maybe we should have amethod on the Domain Object to generate the `meetingID` like `videoConference.getMeetingId()`. The DO schould then probably also not expose its salt directly.

Reply:
I decided to leave it as is, as the DO is not designed to handle that kind of business logic, i.e. not so easy to implement. Could be part of a refactoring task